homework/index.js

```diff
@@ -0,0 +1,64 @@
+function getPromise() {
+  const ret = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+    .then(response => response.json());
+  return ret;
+}
+
+
+function populate(Prom) {
```

Comment:
Parameter names should begin with a small letter; and clearly denote what it contains; in this case e.g. `repoDataPromise`.

Also, from a function with name 'populate', I would rather expect it to populate the select list. For this function, I think `selectRepo` or something would make more sense.

Instead of (or additional to) the promise parameter, I would then expect it to have a parameter with the selected repo name - which you'd pass from the change event listener. Then you can leave off the code to get the value from the DOM.

Reply:
i did it but i am afraid that what i did is hard coding.